### PR TITLE
Added Laracon EU conferences in Madrid and Amsterdam

### PR DESCRIPTION
**Pull request by jeroengerits**

### CODE REVIEW

**Comment by jeroengerits** [comment]
Become a speaker
Submit for Laracon EU Madrid. Submissions deadline is April 5th..
Submit for Laracon EU Amsterdam. Submissions deadline is May 15th..

**Comment by petk** [comment]
Applied via 6e5057d2518a158d509eda3b7158fa706b321062 Thanks @jeroengerits 👍

**Comment by php-pulls** [comment]
**Comment on behalf of petk at php.net:**

Applied as noted above via 6e5057d2518a158d509eda3b7158fa706b321062

File: archive/entries/2019-03-11-1.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<entry xmlns="http://www.w3.org/2005/Atom" xmlns:default="http://php.net/ns/news">
+  <title>Laracon EU 2019 Amsterdam</title>
+  <id>http://php.net/archive/2019.php#id2019-03-11-1</id>
+  <published>2019-03-11T17:26:39+00:00</published>
+  <updated>2019-03-11T17:26:39+00:00</updated>
+  <default:finalTeaserDate xmlns="http://php.net/ns/news">2019-08-28</default:finalTeaserDate>
+  <category term="conferences" label="Conference announcement"/>
+  <link href="http://php.net/conferences/index.php#id2019-03-11-1" rel="alternate" type="text/html"/>
+  <default:newsImage xmlns="http://php.net/ns/news" link="https://laracon.eu/2019/" title="Laracon Logo">laracon-logo.png</default:newsImage>
```
Comment (salathe) [inline archive/entries/2019-03-11-1.xml]: The logo references the Madrid conference in May.  Did you mean to use a different image for the Amsterdam conference announcement?

Comment (salathe) [reply archive/entries/2019-03-11-1.xml]: Nevermind, I should learn to read. The image has both conferences/dates.  😊

**Pull request closed without merging**